Examples/Tutorials/Tutorial-DpdkL2Fwd/main.cpp

```diff
@@ -68,49 +52,46 @@ int main(int argc, char* argv[])
 	pcpp::DpdkDeviceList::initDpdk(coreMaskToUse, MBUF_POOL_SIZE);
 
 	// Find DPDK devices
-	pcpp::DpdkDevice* device1 = pcpp::DpdkDeviceList::getInstance().getDeviceByPort(DEVICE_ID_1);
-	if (device1 == NULL)
+	auto* device1 = pcpp::DpdkDeviceList::getInstance().getDeviceByPort(DEVICE_ID_1);
```

Comment:
In my opinion, I don't believe we should refactor the code with these kinds of "auto" constructs.